### ISSUE TITLE
Starter template doesn't start in iOS

### ISSUE THREAD

**Issue by mirkokiefer** [nandorojo/solito#433]
I tried the tailwind template and launched `yarn native` and then opened the simulator with `i`.

I get this error:

```
CommandError: No development build (com.solito.nativewind) for this project is installed. Please make and install a development build on the device first.
Learn more: https://docs.expo.dev/development/build/
```

**Comment by derek-primumco** [comment]
`yarn native` is no longer the right command; try `yarn ios` (`yarn expo run:ios`)

**Comment by nandorojo** [comment]
`yarn native ` should work, you just need to create a dev client first. or you can remove `—dev-client` from the native command to use expo go, but i recommend the dev client instead by running `yarn ios` first

**Comment by mirkokiefer** [comment]
I figured I had to go into `cd apps/expo` to be ablel to run `yarn ios`. I was using the `yarn native` command in the root folder.

**Comment by nandorojo** [comment]
Yeah I should make this more clear now that SDK 49 is in the starters. I'm thinking we remove `--dev-client` from the `yarn native` script in the root.

**Comment by SVronskiy** [comment]
Try to change root package.json to
```
  "scripts": {
    "android": "cd apps/expo && npx expo run:android",
    "ios": "cd apps/expo && npx expo run:ios",
    "web": "cd apps/next && yarn next"
  },
```
and use Simpulator with iOS 17. It works for me.

**Comment by nandorojo** [comment]
Hey guys, I pushed a fix for the newest starters. They now use Expo Go by default. Expo is very confusing with this workflow tbh, I wish it were more clear.

Option 1: If you're on an existing starter, just remove `--dev-client` from the `start` script in `apps/expo`. This lets you use Expo Go. Unlikely workflow to stick with.

Option 2: If you want to use a dev client (recommended), first run `expo run:ios` or `expo run:android` in `apps/expo`, and _then_ run `yarn native`.
